hubblestack/extmods/modules/nova.py

```diff
@@ -336,3 +367,46 @@ def __get_audit_files(audit_files):
     return ['salt://' + BASE_DIR_NOVA_PROFILES + os.sep + audit_file.replace('.', os.sep) + '.yaml'
                     for audit_file in audit_files]
     
+
+def __evaluate_results(result_dict, combined_dict, show_compliance):
+    """
+    Evaluate the result dictionary to be returned by the audit module
+    :param result_dict:
+    :param combined_dict:
+    :param show_compliance:
+    :return:
+    """
+    for audit_file in combined_dict:
+        result_list = combined_dict[audit_file]
+        for result in result_list:
+            dict={}
+            dict[result['tag']]=result['description']
```

Comment:
I think you missed that attribute which says: "Do not include me in individual result". 
I think that is: "sub_check: True" or sub_module

Reply:
Done